### DISCUSSION TITLE
Auftragsbestätigung BT-14

### DISCUSSION THREAD

**Discussion by stefanbutz** [stephanstapel/ZUGFeRD-csharp#377] [General]
Hallo,
zunächst vielen Dank für die Bereitstellung der sehr nützlichen Bibliothek.

Ich habe eine Frage zu SellerOrderRefercendDocument (BT-14).
Warum wird das nur im Profil Comfort und Extended ausgegeben und nicht im Profil XRECHNUNG?
Ich finde in der Dokumentation nichts, warum man es im Profil XRECHNUNG nicht ausgeben darf, sondern nur, dass es Optional ist.

Vielen Dank vorab.

**Comment by stephanstapel** [comment]
@stefanbutz ,
ja, da hast Du wahrscheinlich recht. Leider finde ich eine vollständige Strukturbeschreibung wie diese hier:
https://portal3.gefeg.com/invoice/tthome/index/617afdc4-623f-44e0-a05b-5b878840e508

lediglich bis Version 2.1.1. Hast Du eine Idee, wo es diese Beschreibungen für aktuelle Versionen gibt?

**Reply by stefanbutz** [reply]
Ich habe das hier gefunden:
https://xeinkauf.de/app/uploads/2024/07/XRechnung-v3.0.2.pdf

Findest du auf der Seite https://xeinkauf.de/xrechnung/versionen-und-bundles/

Mir ist noch etwas aufgefallen:
Bei dir wird erst das BuyerOrderReferencedDocument und dann das SellerOrderReferencedDocument.
Das führt zu einem Validierungsfehler.
Die müssen anders herum ausgegeben werden.

Vielen Dank für die schnelle Antwort.

**Reply by stephanstapel** [reply]
Die Dokumente nutze ich, allerdings fehlt mir eine Struktur-Ansicht, wie sie die GEFEG erzeugt hatte. Ich habe denen gerade mal geschrieben, mal schauen.

Kannst Du einmal die Reihenfolge aller Elemente raussuchen:
* BuyerOrderReferencedDocument
* SellerOrderReferencedDocument
* ContractReferencedDocument
* AdditionalReferencedDocument

bitte mit Quellenangabe?

und hier posten? Dann passe ich das gerne an.

**Reply by stefanbutz** [reply]
Hallo Stephan,

ich bin leider kein Fachmann was XML und Schemas angeht. Ich habe keine graphische Struktir gefunden lediglich sowas:

  <xs:complexType name="HeaderTradeAgreementType">
    <xs:sequence>
      <xs:element name="BuyerReference" type="udt:TextType" minOccurs="0"/>
      <xs:element name="SellerTradeParty" type="ram:TradePartyType"/>
      <xs:element name="BuyerTradeParty" type="ram:TradePartyType"/>
      <xs:element name="SellerTaxRepresentativeTradeParty" type="ram:TradePartyType" minOccurs="0"/>
      <xs:element name="SellerOrderReferencedDocument" type="ram:ReferencedDocumentType" minOccurs="0"/>
      <xs:element name="BuyerOrderReferencedDocument" type="ram:ReferencedDocumentType" minOccurs="0"/>
      <xs:element name="ContractReferencedDocument" type="ram:ReferencedDocumentType" minOccurs="0"/>
      <xs:element name="AdditionalReferencedDocument" type="ram:ReferencedDocumentType" minOccurs="0" maxOccurs="unbounded"/>
      <xs:element name="SpecifiedProcuringProject" type="ram:ProcuringProjectType" minOccurs="0"/>
    </xs:sequence>
  </xs:complexType>


Ist das ein Hinweis auf die Reihenfolge?

**Comment by stephanstapel** [comment]
done
